airgun/entities/ldap_authentication.py

```diff
@@ -53,6 +53,18 @@ def delete(self, entity_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def read_auth_source_counts(self, auth_source_type):
+        """Read values for corresponding auth source card count"""
+        view = self.navigate_to(self, 'All')
+        if auth_source_type.lower() == 'internal':
+            return view.internal.count
+        elif auth_source_type.lower() == 'external':
+            return view.external.count
+        elif auth_source_type.lower() == 'ldap':
+            return view.ldap.count
+        else:
```

Comment:
This can be replaced with attribute lookup:
```python
return getattr(view, auth_source_type.lower()).count
```
`getattr` will raise `AttributeError` if you try to access attribute that does not exist. You can catch it and raise your `ValueError` if that's important for you.

Reply:
thanks, let me know I was trying figure it out how to do this. that was the reason I kept the name as same.